### DISCUSSION TITLE
peak set before iterative merging across clusters

### DISCUSSION THREAD

**Discussion by AnjaliC4** [GreenleafLab/ArchR#1301] [Questions / Documentation] [answered]
Hi @rcorces,

I used addReproduciblePeakSet function to add peaks per cluster. I know we can extract the final iteratively merged peak set across clusters from the data. But, Is it possible to get peaks that were iteratively merged to form a cluster-specific peak set and not those that were iteratively merged across the clusters (please see the image). I would like to do some model training with the peak set derived before the iterative merging process across the clusters. There must be a way to obtain this information from all the metadata stored in ArchR project? Thanks

Peaks right before this step ->

<img width="827" alt="Screen Shot 2022-02-21 at 1 36 53 PM" src="https://user-images.githubusercontent.com/60046859/155010693-da40a045-7a6f-4bf9-95f0-63dc03ae99b3.png">

**Comment by AnjaliC4** [comment, marked as answer]
On further digging, I found that under PeakCalls folder in the project there are: Cluster1-Sample-summits.rds for all the samples and Cluster1-reproduciblePeaks.gr.rds (for all the clusters). I believe Cluster1-reproduciblePeaks.gr.rds are peaks iteratively merged in a cluster and not merged across the clusters. Please let me know if my understanding is correct. Thanks

**Reply by rcorces** [reply]
correct
